Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -1270,7 +1270,7 @@ lemma addMap_of_Z_ne_zero {P Q : Fin 3 → F} (hP : W.Equation P) (hQ : W.Equati
 
 lemma nonsingularLift_addMap {P Q : PointClass F} (hP : W.NonsingularLift P)
     (hQ : W.NonsingularLift Q) : W.NonsingularLift <| W.addMap P Q := by
-  rcases P, Q with ⟨⟨_⟩, ⟨_⟩⟩
+  rcases P; rcases Q
```

Comment:
This syntax is nice and new to me but isn't shorter here.

If this commit looks good let's send the PR to maintainers.

Reply:
LGTM!